src/main/java/uk/ac/ucl/rc/development/oracc/ext/TextLineNumber.java

```diff
@@ -229,7 +247,6 @@ private void setPreferredWidth()
 			Dimension d = getPreferredSize();
 			d.setSize(preferredWidth, HEIGHT);
 			setPreferredSize( d );
-			setSize( d );
```

Comment:
How come this was removed? Did it cause a problem or do nothing?

Reply:
If I remember correctly, it was doing nothing.  See https://stackoverflow.com/q/1783793/2442087

Reply:
Ah, okay. The surrounding `JPanel` (the AtfAreaView) uses a `BorderLayout` manager, so it looks like we're okay just using `setPreferredSize`.